e2e/cypress/fixtures/launch-darkly-flags.json

```diff
@@ -0,0 +1,3 @@
+{
+  "validation": true
```

Comment:
@thetif are we keeping this flag? The admin check stuff is using `adminCheck` flag

Reply:
I also had this question, since admin check technically turns on the validation